### ISSUE TITLE
Android 12 incompatibility

### ISSUE THREAD

**Issue by DarkPhilosophy** [Tu2l/AnimeBoya#32]
I am using samsung on android 12 and it keep crashing everytime I open apk

**Comment by kloklojul** [comment]
Yep same. Updated to one ui 4 and Android 12 and it no longer works

**Comment by GABIlopez** [comment]
https://github.com/Killerpac/Kayuri/releases/download/v1.5.0/Kayuri-v1.5.0-arm64-v8a-release.apk
